tests/integration/cqlengine/model/test_udts.py

```diff
@@ -286,3 +286,43 @@ class AllDatatypesModel(Model):
 
         for i in range(ord('a'), ord('a') + 15):
             self.assertEqual(input[chr(i)], output[chr(i)])
+
+    def test_nested_udts_inserts(self):
+        """
+        Test for inserting collections of user types using cql engine.
+        test_nested_udts_inserts Constructs a model that contains a list of usertypes. It will then attempt to insert
+        them. The expectation is that no exception is thrown during insert. For sanity sake we also validate that our
+        input and output values match.
+
+        @since 2.6.0
+        @jira_ticket PYTHON-311
+        @expected_result. No syntax exception thrown
+
+        @test_category data_types:udt
+        """
+
+        # This combination of model, and UT produces a syntax error in 2.5.1 due to improper quoting around
+        # the names collection
```

Comment:
Not sure why this inline comment is here. If this info is needed, it should be in the comment-block description.